easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,46 @@
+name = 'OpenMPI'
+version = '4.1.1'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = [
+    'OpenMPI-4.1.1_fix-bufferoverflow-in-common_ofi.patch',
+    'OpenMPI-4.0.6_remove-pmix-check-in-pmi-switch.patch',
+    '9212.patch',
+    '9213.patch',
```

Comment:
We should rename these patch files, to start with `OpenMPI-4.1.1`, and have a more descriptive name.

Where they come from should be mentioned in the patch header (a link to the PR), not in the filename.